code/README.md

```diff
@@ -1 +1,101 @@
-The subdirectory `hello` is used to check that your machine is correctly installed.
+Exercises HowTo
+===============
+
+List of exercises per day, with preferred order and solution overview.
+Each exercise is in its own directory and referred to in the following by the name of the directory
+For each day, the exercises are given in order in which they should be done.
+
+Setup requirements
+------------------
+
+### Required
+
+ - decent C++ editor
+ - any C++ compiler supporting C++ 20
+ - C++ 17 compiler will do for most of the exercises though
+ - git for getting/managing your code
+
+### Good to have
+
+ - gdb to debug your problems
+ - valgrind, kcachegrind, cppcheck, ... for corresponding exercises
+
+### C++ and python specific needs
+
+ - python3, libpython3-dev
+ - ctypes, matplotlib, numpy python packages
+
+
+Getting exercises’ code
+-----------------------
+
+git clone https://github.com/hsf-training/cpluspluscourse.git
+cd cpluspluscourse/code
+
+
+Anatomy of an exercise
+----------------------
+
+Each exercise is in a subdirectory with mainly 3 sets of files
+
+### *.hpp and *.cpp files
+
+  - the code to understand and fix/complete
+  - you never start from scratch
+
+### Makefile / cmake
+
+  - prepared Makefile for easy compilation
+  - “make” and “make clean” are available
+  - cmake is also supported
+
+### solution subdirectory
+
+  - the solution to the exercise (“make solution”)
+  - please do not use before trying !
+
+Just follow the instructions
+----------------------------
+
+ - Each exercise comes with a set of instructions in the course
+ - Also present in exercise subdir in markdown format
+ - Very practical to read on github
+
+How to test your setup
+----------------------
+
+### Just use hello exercise
+
+ - go to code/hello
+ - follow the README
+
+### Practically for compilation
+
+  - cd code/hello
+  - make;
+  - export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:.
+  - ./hello
+
+### Practically for valgrind/cppcheck
+
+  - valgrind --tool=callgrind ./hello; kcachegrind
+  - cppcheck .
+
+Using lxplus
+------------
+
+  - lxplus is perfectly suitable for the exercises
+  - is has all the needed compilers/tools in cvmfs
+
+### Practically on lxplus8
```

Comment:
So, would you merge the Windows guide into this README as well?

Reply:
Once this is merged, I would say yes. At this stage, I'm fighting md for the schedule files. And gitlab md is special and cannot relaly be debugged locally... a pleasure :-)